cmd/security-agent/app/status.go

```diff
@@ -68,7 +68,7 @@ func requestStatus() error {
 	var e error
 	var s string
 	c := util.GetClient(false) // FIX: get certificates right then make this true
-	urlstr := fmt.Sprintf("https://localhost:%v/agent/status", config.Datadog.GetInt("compliance_config.cmd_port"))
+	urlstr := fmt.Sprintf("https://localhost:%v/agent/status", config.Datadog.GetInt("security_agent_config.cmd_port"))
```

Comment:
We may have a conflicting change, let's use `security_agent.cmd_port` for this.

I'll rebase https://github.com/DataDog/datadog-agent/pull/6112 if your PR goes in first.